<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormContainerQuery</class>
 <widget class="QWidget" name="FormContainerQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>箱号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_CTNNO"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>箱型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_CTNTYPE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_CTNPOS"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>尺寸</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_CTNSIZE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>箱主</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_CTNOWNER"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="queryButton">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
